tests/test_stl_binders.cpp

```diff
@@ -10,6 +10,7 @@
 #include <pybind11/numpy.h>
 #include <pybind11/stl_bind.h>
 
+#include "pybind11/detail/type_caster_base.h"
```

Comment:
Could you please remove this include? To not encourage others to include a `detail` header. — The pybind11 code organization has me wishing a lot. Improving the organization will be harder if we implicitly suggest that the detail headers are part of the public interface.

Reply:
Sure, I accidentally left it there before committing.